addons/test/testutil/test_helpers.go

```diff
@@ -42,6 +42,63 @@ import (
 	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/webhooks"
 )
 
+type WebhookCertificatesDetails struct {
```

Comment:
@blc1996 , My PR is already merged, so I am wondering why this change is showing up here in your diff

Reply:
I move it up a bit